Userland/Libraries/LibJS/Tests/builtins/WeakSet/WeakSet.prototype.add.js

```diff
@@ -0,0 +1,26 @@
+test("basic functionality", () => {
+    expect(WeakSet.prototype.add).toHaveLength(1);
+
+    const weak_set = new WeakSet([{ a: 1 }, { a: 2 }, { a: 3 }]);
+    expect(weak_set.add({ a: 4 })).toBe(weak_set);
+    expect(weak_set.add({ a: 1 })).toBe(weak_set);
+});
+
+test("invalid values", () => {
+    const weak_set = new WeakSet();
```

Comment:
Please use pascalCase in all JS code - same below.

Reply:
fixed!

Reply:
oh whoops, i messed up the commit, just a sec.